app/containers/message/Message.js

```diff
@@ -63,11 +63,15 @@ MessageInner.displayName = 'MessageInner';
 const Message = React.memo((props) => {
 	if (props.isThreadReply || props.isThreadSequential || props.isInfo) {
 		const thread = props.isThreadReply ? <RepliedThread {...props} /> : null;
+		const { username } = props.author;
 		return (
 			<View style={[styles.container, props.style]}>
 				{thread}
 				<View style={[styles.flex, styles.center]}>
 					<MessageAvatar small {...props} />
+					<Text style={styles.username}>
```

Comment:
`style={[styles.username, { color: themes[theme].color }]}`

Reply:
My bad, I've chosen the titleText